lib/Internal/Posix/Runner.php

```diff
@@ -84,6 +84,8 @@ public function start(string $command, string $cwd = null, array $env = [], arra
             $message = "Could not start process";
             if ($error = \error_get_last()) {
                 $message .= \sprintf(" Errno: %d; %s", $error["type"], $error["message"]);
+            } elseif (!function_exists('proc_open')) {
+                $message .= "because proc_open() does not exist. Make sure its not disabled via 'disable_functions' in your php.ini";
```

Comment:
An error is thrown if the function doesn't exist: https://3v4l.org/Z7jp5

Reply:
proc_open virtually exists but is not allowed to called from userland.